host/sgx/create.c

```diff
@@ -1006,7 +1035,7 @@ oe_result_t oe_sgx_build_enclave(
         debug_enclave->path = enclave->path;
         debug_enclave->path_length = strlen(enclave->path);
 
-        debug_enclave->base_address = (void*)enclave->addr;
+        debug_enclave->base_address = (void*)enclave->start_address;
```

Comment:
Can you create an issue to rename base_address to start_address in oe_debug_enclave_t?
That issue does not have to be part of the zero-base meta-issue; it is an optional item.

Reply:
Created issue #4135.